homeassistant/components/bayesian/manifest.json

```diff
@@ -1,7 +1,9 @@
 {
   "domain": "bayesian",
   "name": "Bayesian",
+  "after_dependencies": ["schedule"],
```

Comment:
This is required by hassfest.
All the other imported components are specifically excused from being required as dependencies. Most because they are Platforms. Schedule is not and so must be listed as a dependency. Schedule is important to keep as it may be used by users to estimate the likelihood of presence or something happening like "working".

https://github.com/home-assistant/core/blob/cb9cc0f801118ae73e2cef959fdec274cd645293/script/hassfest/dependencies.py#L118-L125

Reply:
Done in eacb6be